src/SpotifyUsername.jsx

```diff
@@ -28,10 +28,12 @@ const SpotifyUsername = () => {
       <div className="profileContainer">
         <div className="imageCropper">
           {user.profileImage ? (
-            <img
-              className="profileImg"
-              src={user.profileImage}
-              alt="your profile pic"
+            <div
+              className="center-cropped"
+              style={{
```

Comment:
Try to reimplement it as img tag without dynamic styles - just with a single css class name.
Hint: use https://developer.mozilla.org/ru/docs/Web/CSS/object-fit